openvpn-install.sh

```diff
@@ -214,9 +214,16 @@ function installQuestions() {
 	echo "I need to ask you a few questions before starting the setup."
 	echo "You can leave the default options and just press enter if you are ok with them."
 	echo ""
+
+	echo "Please provide the VPN network prefix. If you are going to use  10.8.0.0/24, then type prefix 10.8.0"
```

Comment:
I don't see why this question should be the first one to be asked.

Reply:
Thanks for your comments.
Any order is ok. Which order do you consider proper?
I just need a way to setup the IP Range, if PR #550 allows this, then fair enough.